### PR TITLE
Add rest api config for proto

### PR DESCRIPTION
**Pull request by coderbradlee**
1、curl -X GET localhost:8081/v1/getAccount?address=io1vdtfpzkwpyngzvx7u2mauepnzja7kd5rryp0sg

2、curl -X POST --data '{"byIndex":{"start":1,"count":1}}' localhost:8081/v1/getActions

3、curl -X POST --data '{"byIndex":{"start":1,"count":1}}' localhost:8081/v1/getBlockMetas

4、curl -X GET localhost:8081/v1/getChainMeta

5、curl -X GET localhost:8081/v1/getServerMeta

6、curl -X  POST --data '{"action": {"core": {"version": 1, "nonce": 2, "gasLimit": 10000, "gasPrice": "10", "transfer": {"amount": "100", "recipient": "io1sxm6zl56um2c3ntq5fwqjar4za5ka560x53muy"}}, "senderPubKey": "BOk7WxyPumkmNlKkg61VMY5O7VtRIjFMt/2wd9jHKVCXzsku5QsRCNx0lalyDlkh5W0wSON6vmpnFtfJuRPp8uY=", "signature": "9mrqFBggiRocZhkRVUswxs83NaEFNdEYYczI8049vlovHEP4YMQz+3Isznc3CrYaJxAbc2PTIz7y2meerJ8bHAA="}}' localhost:8081/v1/sendAction

7、curl -X GET localhost:8081/v1/getReceiptByAction?actionHash=4ea1cbec4fd3b9cbfa95ec26fafe7c772497d50c090f17eedb55a504f09f3873

8、curl -X  POST --data '{"execution":{"amount":"0","contract":"io1hhu3gwt5uankzl3zlp2cz8w0sl9uj336rq0334","data":"Bv3eAw=="}, "callerAddress": "io1vdtfpzkwpyngzvx7u2mauepnzja7kd5rryp0sg"}' localhost:8081/v1/readContract 

9、curl -X GET localhost:8081/v1/suggestGasPrice

10、curl -X  POST --data '{"action": {"core": {"version": 1, "nonce": 2, "gasLimit": 10000, "gasPrice": "10", "execution": {"amount": "0", "contract": ""}}, "senderPubKey": "BOk7WxyPumkmNlKkg61VMY5O7VtRIjFMt/2wd9jHKVCXzsku5QsRCNx0lalyDlkh5W0wSON6vmpnFtfJuRPp8uY=", "signature": "9mrqFBggiRocZhkRVUswxs83NaEFNdEYYczI8049vlovHEP4YMQz+3Isznc3CrYaJxAbc2PTIz7y2meerJ8bHAA="}}' localhost:8081/v1/estimateGasForAction

11、curl -X  POST --data '{"transfer":{"amount":"1","recipient":"1","payload":""},"callerAddress":"io1vdtfpzkwpyngzvx7u2mauepnzja7kd5rryp0sg"}' localhost:8081/v1/estimateActionGasConsumption

12、curl -X GET localhost:8081/v1/readState?protocolID=cmV3YXJkaW5n&methodName=VW5jbGFpbWVkQmFsYW5jZQ==&arguments=aW8xanV2eDVnMDYzZXU0dHM4MzJudWtwNHZnY3drMmduYzVjdTlheWQ=

13、curl -X GET localhost:8081/v1/getEpochMeta?epochNumber=1

14、curl -X GET localhost:8081/v1/getRawBlocks?startHeight=100&count=1&withReceipts=true

15、curl -X POST --data '{"filter":{"address":["io1hhu3gwt5uankzl3zlp2cz8w0sl9uj336rq0334"]},"byBlock":{"blockHash":"pMJI2EC6t+zb3gWRSKKJAhzhVGYbMQfaZ0dOK/pCOOg="}}' localhost:8081/v1/getLogs

16、curl -X GET localhost:8081/v1/getVotes?votee=726f626f7462703030303136&height=8164200&offset=0&limit=2

17、curl -X GET localhost:8081/v1/streamBlocks

18、curl -X POST --data '{"filter":{"address":["io1hhu3gwt5uankzl3zlp2cz8w0sl9uj336rq0334"]}}' localhost:8081/v1/streamLogs

2、3、6、8、10、11、15、18 need to add filters when write grpc-gateway server

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,8 +17,9 @@ GOGET=$(GOCMD) get
 
 .PHONY: gogen
 gogen:
-	@protoc --go_out=plugins=grpc:${GOPATH}/src ./proto/types/*
-	@protoc --go_out=plugins=grpc:${GOPATH}/src ./proto/rpc/*
-	@protoc --go_out=plugins=grpc:${GOPATH}/src ./proto/testing/*
-	@protoc -I. -I./proto/types --go_out=plugins=grpc:${GOPATH}/src ./proto/api/*
-	@protoc -I. --grpc-gateway_out=logtostderr=true:${GOPATH}/src ./proto/api/*
+	@protoc -I. -I${GOPATH}/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis --go_out=plugins=grpc:${GOPATH}/src ./proto/types/*
```
Comment (dustinxie) [inline Makefile:20]: is there any change to api.pb.go after rerun the gogen script?

**Review by dustinxie — COMMENTED** [review]
one comment

File: Makefile (hunk 1 above)
Comment (coderbradlee) [reply Makefile:20]: a lot of changes to api.pb.go and api.pb.gw.go,update to this pr?

**Comment by coderbradlee** [comment]
outdated

**Pull request closed without merging**